masterlist.yaml

```diff
@@ -1639,6 +1675,56 @@ plugins:
   - name: 'Flight of the ValkED-E.esp'
     tag:
       - Sound
+  - name: 'Equipment Restoration Project.esp'
+    url: [ 'http://www.nexusmods.com/newvegas/mods/43875' ]
+    inc:
+      - 'WeaponModsExpanded.esp'
+      - 'Weapon Mod Expansion.esp'
+      - 'Weapon Mod Expansion.esm'
+    after:
+      - 'EVE FNV - ALL DLC.esp'
+      - 'IMPACT.esp'
+      - 'IMPACT - ALLDLC.esp'
+      - 'IMPACT-POPMerged.esp'
+      - 'jsawyer.esp'
+      - 'JSawyer - CE.esp'
+      - 'Project Nevada - Cyberware Additions.esp'
+      - 'Project Nevada - All DLC.esp'
+	  - '357retex.esp'
```

Comment:
Make sure you're not mixing tab indentation and space indentation. (We use space indentation for the `masterlist.yaml` files.)